src/mumble/MainWindow.h

```diff
@@ -146,6 +146,9 @@ class MainWindow : public QMainWindow, public Ui::MainWindow {
 	void focusNextMainWidget();
 	QPair< QByteArray, QImage > openImageFile();
 
+	void applyState(bool minimalView);
```

Comment:
```suggestion
	void restoreState(bool minimalView);
```
That seems like a more descriptive name to me

Reply:
It's the same as Qt's though.

Reply:
You mean the one I proposed? Shouldn't matter as it will have a different function signature. There might be some confusion though 🤔

However, `apply` implies (to me) that it gets an argument that it should apply which is jot the case.

Maybe `loadState` as a counterpart to `saveState`?

Reply:
Sounds good.

Reply:
Please note that the counterpart is called "storeState`.

Reply:
Please note that the counterpart is called `storeState`.